src/WooCommerceSalutation.php

```diff
@@ -65,6 +65,7 @@ public static function woocommerce_default_address_fields(?array $fields): array
         'Mrs' => __('Mrs', Plugin::L10N),
         'Mr' => __('Mr', Plugin::L10N),
         'Company' => __('Company', Plugin::L10N),
+        'Divers' => __('Divers', Plugin::L10N),
```

Comment:
We should probably also move this before Company, otherwise the association with a gender gets lost.

Reply:
Updated in the last commit.